Gruntfile.js

```diff
@@ -286,6 +289,11 @@ module.exports = function(grunt) {
     nodeunit: {
       all: ['tests/nodeunit/unit/**/*.js']
     },
+    mochaTest: {
+      integration: {
+        src: ['tests/integration/**/*.js'],
+      },
+    },
```

Comment:
> Mocha is… a better nodeunit?

Or at least a less-bad one.  Our karma tests use mocha as well, so I've not added a new framework, just a new grunt integration.